src/peerconnection.cpp

```diff
@@ -140,6 +140,11 @@ void PeerConnection::setLocalDescription(Description::Type type) {
 	if (!iceTransport)
 		return; // closed
 
+	if (init.iceUfrag && init.icePwd) {
+		PLOG_DEBUG << "Using custom ufrag/pwd";
```

Comment:
You could print the values.
```suggestion
		PLOG_DEBUG << "Using custom ICE attributes, ufrag=\"" << init.iceUfrag.value() << "\", pwd=\"" << init.icePwd.value() << "\"";
```